src/main/java/eu/mihosoft/monacofx/SystemClipboardWrapper.java

```diff
@@ -0,0 +1,67 @@
+package eu.mihosoft.monacofx;
+
+import javafx.application.Platform;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyCodeCombination;
+import javafx.scene.input.KeyCombination;
+import javafx.scene.input.KeyEvent;
+
+/**
+ * Wrapper class to make the clipboard functionality testable.
+ */
+public class SystemClipboardWrapper {
+
+	private final KeyCodeCombination KEY_CODE_CTRL_C = new KeyCodeCombination(KeyCode.C, KeyCombination.CONTROL_DOWN);
```

Comment:
Is KeyCombination.CONTROL_DOWN the correct choice here? I am not 100% sure, but I believe in order to work on a MAC keyboard what should be used is [KeyCombination.SHORTCUT_DOWN](https://docs.oracle.com/javase/10/docs/api/javafx/scene/input/KeyCombination.html#SHORTCUT_DOWN)

Reply:
I am not sure. I can try on a Mac though. But I see nothing that prevents us from merging. Thanks @kia for doing the work!

Reply:
You are welcome @miho !

Reply:
@MegaCoderX thanks for the hint. There is a pull request now with the change you suggested.